src/ceph_fuse.cc

```diff
@@ -143,7 +143,12 @@ int main(int argc, const char **argv, const char *envp[]) {
 	  char buf[5050];
 	  string mountpoint = cfuse->get_mount_point();
 	  snprintf(buf, 5049, "fusermount -u -z %s", mountpoint.c_str());
-	  system(buf);
+	  int umount_r = system(buf);
+	  umount_r = WEXITSTATUS(umount_r);
+	  if (umount_r) {
+	    cerr << "got error " << cpp_strerror(umount_r)
```

Comment:
umount_r the exit status at this point will always be 1 on error, so printing this as an errno string is no appropriate.

"fusermount returns 0 for successful operation, or non-zero if one of the
     operations did not complete successfully."

I verified from the fusermount source code it exits with status 0 on success and 1 on error.